packages/manager/apps/dedicated/client/app/ip/byoip/byoip.controller.js

```diff
@@ -0,0 +1,57 @@
+import get from 'lodash/get';
+import { AS_OPTIONS, IPV4_BLOCK_PATTERN } from './byoip.constants';
+
+export default class IpByoipConfiguration {
+  /* @ngInject */
+  constructor($translate, Alerter, ByoipService) {
+    this.$translate = $translate;
+    this.Alerter = Alerter;
+    this.ByoipService = ByoipService;
+
+    this.AS_OPTIONS = AS_OPTIONS;
+    this.IPV4_BLOCK_PATTERN = IPV4_BLOCK_PATTERN;
+  }
+
+  $onInit() {
+    this.byoip = {};
+    this.ipRir = get(this.plan, 'details.product.configurations').find(
```

Comment:
don't know if it's necessary here but in order to handle undefined result without raising an error it's safer to use :
```
get(this.plan, 'details.product.configurations', []).find
```